front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PermissionsController.groovy

```diff
@@ -72,6 +72,12 @@ public class PermissionsController {
     return actualPermissions.values()
   }
 
+  @ApiOperation(value = "", notes = "Get specific application permissions. Internal use only.")
```

Comment:
I think technically all Front50 APIs are "internal use only" so I would just remove that.